src/v/kafka/server/group.cc

```diff
@@ -2848,14 +2848,43 @@ ss::future<> group::do_try_abort_old_tx(model::producer_identity pid) {
         }
     }
 
-    std::optional<model::tx_seq> tx_seq;
-
     auto p_it = _prepared_txs.find(pid);
     if (p_it != _prepared_txs.end()) {
-        tx_seq = p_it->second.tx_seq;
-    }
+        auto r = co_await _tx_frontend.local().try_abort(
+          model::partition_id(0), // Partition_id 0???
+          pid,
+          p_it->second.tx_seq,
+          1000ms);
```

Comment:
let's use at least `_sync_timeout(config::shard_local_cfg().rm_sync_timeout_ms.value())` config value just like we do in rm_stm